src/api/api.ts

```diff
@@ -0,0 +1,10 @@
+export const IMDB_URL = 'https://www.imdb.com/title/';
+
+const MOVIES_URL = 'http://www.omdbapi.com/?apikey=c4ea428d&t=';
```

Comment:
const MOVIES_URL = 'http://www.omdbapi.com/?apikey=c4ea428d&t=';
```suggestion
const MOVIES_URL = 'https://www.omdbapi.com/?apikey=c4ea428d&t=';
```

Reply:
fixed